pkg/cloud/gcp/actuators/machine/reconciler.go

```diff
@@ -35,6 +36,7 @@ func newReconciler(scope *machineScope) *Reconciler {
 
 // Create creates machine if and only if machine exists, handled by cluster-api
 func (r *Reconciler) create() error {
+	defer r.reconcileMachineWithCloudState()
```

Comment:
Can create be called multiple times for `r`? If so, what happens with the multiple `defer` calls?